packages/components/src/components/tab/tab.stories.ts

```diff
@@ -33,129 +46,61 @@ export const Default = {
-      },
-      args
-    });
-  }
+export const Variant = {
+  name: 'Variant',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-tab variant="default">Default</sd-tab>
+      <sd-tab variant="container">Container</sd-tab>
+    </div>
+  `
 };
 
 /**
- * Use the variant attribute to alternate between the `default` and `container` styles.
+ * Use the `active` attribute to toggle the active state.
  */
 
-export const Variant = {
-  parameters: { controls: { exclude: ['variant'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: { type: 'attribute', name: 'variant' },
-        x: { type: 'attribute', name: 'active', values: [false, true] }
-      },
-      args,
-      constants: { type: 'attribute', name: 'active', value: true }
-    });
-  }
+export const Active = {
+  name: 'Active',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-tab active>Default</sd-tab>
+      <sd-tab variant="container" active>Container</sd-tab>
+    </div>
+  `
 };
 
 /**
- * Use the `disabled` attribute to toggle the disabled state.
+ * Use the `disabled` attribute to disable a tab.
  */
-
 export const Disabled = {
-  parameters: { controls: { exclude: ['disabled'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: { type: 'attribute', name: 'disabled' }
-      },
-      args
-    });
-  }
+  name: 'Disabled',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-tab disabled>Default</sd-tab>
+    </div>
+  `
 };
 
-export const Parts = {
-  parameters: {
-    controls: {
-      exclude: ['base', 'active-tab-indicator', 'bottom-border']
-    }
-  },
-  render: (args: any) => {
-    return html`
-      ${['base', 'active-tab-indicator', 'hover-bottom-border'].map(part =>
-        generateTemplate({
-          axis: {
-            x: {
-              type: 'template',
-              name: 'sd-tab::part(...){outline: solid 2px red}',
-              values: [
-                {
-                  title: part,
-                  value: `<style>#part-${part} sd-tab::part(${part}){outline: solid 2px red; ${
-                    part === 'tabs' && 'outline-offset:-2px;'
-                  }}</style><div id="part-${part}">%TEMPLATE%</div>`
-                }
-              ]
-            },
-            y: { type: 'attribute', name: 'active', values: [false, true] }
-          },
-          constants: [
-            {
-              type: 'template',
-              name: 'width',
-              value: `
-                <div style="width: 600px; position: relative;">%TEMPLATE%
-                </div>
-              `
-            }
-          ],
-          args
-        })
-      )}
-    `;
-  }
-};
 /**
- * Use the `left` slot to optionally include an element (eg. icon) positioned to the left of the label.
+ * Use the `left` slot include an element (eg. icon) positioned to the left of the label.
  */
+export const WithIcon = {
+  name: 'With Icon',
```

Comment:
[Icon](name: 'Icon',)

Use the ”icon-left” slot to add system icons.

Reply:
<img width="615" alt="Bildschirmfoto 2024-09-03 um 14 47 07" src="https://github.com/user-attachments/assets/eaeeee0f-8c07-4985-9410-e1ad2555e947">
Pls open up a bug issue - the gap between icon and label is missing

Reply:
Pls remove container variant here

Reply:
> [Icon](name: 'Icon',)
> 
> Use the ”icon-left” slot to add system icons.

In this component the slot for the icon is named `left`. If we want to update it to `icon-left` we need to open an issue.

> <img alt="Bildschirmfoto 2024-09-03 um 14 47 07" width="615" src="https://private-user-images.githubusercontent.com/41894356/363987283-eaeeee0f-8c07-4985-9410-e1ad2555e947.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUzNzQ5NjIsIm5iZiI6MTcyNTM3NDY2MiwicGF0aCI6Ii80MTg5NDM1Ni8zNjM5ODcyODMtZWFlZWVlMGYtOGMwNy00OTg1LTk0MTAtZTFhZDI1NTVlOTQ3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTAzVDE0NDQyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZhMjgxMmRkNzc3ZDI3ZjQ3OTJkODEwOGFjYjk5ZDg1ZWE5NjZmMWM5NDY4ZjMxNGNmYmIwYWZlNmU4MGU0OWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.VQe64Kh4GOkUqTZiPpbUgQWepmqfJGFLMeXJflFBgEA"> Pls open up a bug issue - the gap between icon and label is missing

[Bug issue](https://github.com/solid-design-system/solid/issues/1345)

> Pls remove container variant here

Removed

Reply:
Alright we’ll keep "left"- Thanks